src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -129,7 +129,8 @@ internal enum SerializationMode
         {
             CodeGenOnly,
             ReflectionOnly,
-            ReflectionAsBackup
+            ReflectionAsBackup,
+            PreGenOnly
```

Comment:
People will be confused with `CodeGenOnly` vs `PreGenOnly`. By `PreGenOnly`, do you really mean disable RBS and disable RefEmit, or do you mean RBS and RefEmit are backup?